app/routes/docs.contributing.github/route.mdx

```diff
@@ -0,0 +1,120 @@
+---
+handle:
+  breadcrumb: GitHub
+---
+
+import copyUrl from './copy-url.png'
+import githubFork from './github-fork.png'
+import githubImage from './github.png'
+import newFork from './new-fork.png'
+
+# GitHub
+
+GCN consists of several open-source projects grouped into the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization. These include:
+
+- [nasa-gcn/gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov): This web site
+- [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn.nasa.gov): The GCN Notices unified schema
+- [nasa-gcn/gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python), [nasa-gcn/gcn-kafka-js](https://github.com/nasa-gcn/gcn-kafka-js): Kafka client libraries
+  ...
+
+To contribute to any project in the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization, follow these steps:
+
+- [Fork a copy of our repo](#forking-a-repo)
+
+- [Clone your repo to your machine](#cloning-a-repo)
+
+- [Create a branch for your new changes](#branching-off-main)
+
+- [Create a Pull Request to contribute to the original project](#pull-requests)
+
+You will need to install Git on your development machine. Check out [the official installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+## Forking a Repo
+
+Navigate to the page repository page for whichever project you wish to contribute to, for example: [gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov).
+
+<img
+  src={githubImage}
+  width="712"
+  height="392"
+  className="height-auto"
+  alt="gcn.nasa.gov GitHub page"
+/>
+
+Click the Fork button.
+
+<img
+  src={githubFork}
+  width="712"
+  height="392"
+  className="height-auto"
+  alt="Highlighted fork button"
+/>
+
+Click Create Fork to create a new repo in your account.
+
+<img
+  src={newFork}
+  width="712"
+  height="392"
+  className="height-auto"
+  alt="Highlighted new fork"
+/>
+
+## Cloning a Repo
+
+We recommend checking out [this guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on adding an SSH key to your github account.
```

Comment:
Can we put this, along with the Git installation instructions, in a "Setup" or "Prerequisites" section?